helpers/skip_messages/skip_messages.go

```diff
@@ -12,6 +12,7 @@ NOTE: Ensure CNB lifecycle is enabled on your platform before enabling this test
 const SkipFileBasedServiceBindingsBuildpackApp = `Skipping this test because config.IncludeFileBasedServiceBindings is set to 'false'.`
 const SkipFileBasedServiceBindingsCnbApp = `Skipping this test because config.IncludeFileBasedServiceBindings and/or config.IncludeCNB are set to 'false'.`
 const SkipFileBasedServiceBindingsDockerApp = `Skipping this test because config.IncludeFileBasedServiceBindings and/or config.IncludeDocker are set to 'false'.`
+const SkipIncludeIpv6 = `Skipping this test because config.IncludeIpv6 is set to 'false'.`
```

Comment:
I suggest calling this `SkipIPv6`, which makes it clearer what is actually skipped.